### PR TITLE
[TESTING: axe-e2e] Script to automate multi-page accessibility scan

### PR DESCRIPTION
**Pull request by 1Copenut**
### Purpose
Provide teams with a simple way to scan multiple pages for web accessibility errors, as detected by the [axe-core.js](https://github.com/dequelabs/axe-core) library. Users can opt to see errors on their terminal window, or outputted to a CSV report for easy import into a spreadsheet program.

This script may be superseded by a more robust checker using Nightwatch.js later on; it is meant to aid in spot-checking accessibility health of a project.

### Items of Note
* Script makes use of a forked version of the [axe-reports](https://www.npmjs.com/package/axe-reports) library. I wanted to allow console output of errors to encourage more frequent accessibility testing. [View the forked version here](https://github.com/1Copenut/axe-reports). This library uses the MIT license.
* This script is set up to use the Chrome browser in a headless configuration. In anecdotal testing, it has been the fastest way to evaluate large numbers of pages.

### CODE REVIEW

File: test-scripts/axe-e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# axe-e2e
+A simple script to analyze multiple pages for accessibility errors, and output
+them to a terminal window, or CSV file. The script runs automatically, and is
+designed to evaluate large numbers of pages.
+
+## Usage
+1. Add the `/axe-e2e/` directory to your project under test
```
Comment (dannychapman) [inline test-scripts/axe-e2e/README.md]: Could you clarify this, I'm not sure what `under test` means here.

Comment (1Copenut) [reply test-scripts/axe-e2e/README.md]: Yeah, that could have been worded better. Just pushed an update.

**Comment by 1Copenut** [comment]
@dannychapman Updated the documentation, added a top-level argument to pass in selector ID, and added ChromeDriver to the package.json file, so a local instance can be added and NPM will figure out the path dependency.

**Comment by dannychapman** [comment]
@1Copenut LGTM!

**Pull request merged**